src/components/fields/Email/Settings.tsx

```diff
@@ -0,0 +1,33 @@
+import { ISettingsProps } from "@src/components/fields/types";
+import { TextField, Button } from "@mui/material";
+import { useRef } from "react";
+
+export default function Settings({ onChange, config }: ISettingsProps) {
+
+    const emailFieldValidation = useRef<HTMLInputElement>(null);
+    const useStandardRegex = () => {
+        if (emailFieldValidation.current != null) {
+            emailFieldValidation.current.value = "^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+.[a-zA-z]{2,3}$";  
```

Comment:
```suggestion
            onChange("validationRegex")("^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+.[a-zA-z]{2,3}$");
```

As I wrote in a [previous comment](https://github.com/rowyio/rowy/pull/820#issuecomment-1264966127):

> it works when I manually update the regex (such as adding a space at the end) then saving.

> it’s only set when there is an input to the text field, so it has to be typed in first.

This is because you call `onChange` when the user types in the text field only. You manually change the value of the input element in the DOM, which doesn’t update the React state.

So when the user clicks the button, it updates the rendered input but doesn’t save in the column config, so it will never validate unless if they type in the input _after_ clicking the button.